packages/execute-driver/lib/execute-child.js

```diff
@@ -0,0 +1,141 @@
+import _ from 'lodash';
+import B from 'bluebird';
+import { NodeVM } from 'vm2';
+import { logger, util } from '@appium/support';
+import { attach } from 'webdriverio';
+
+const log = logger.getLogger('ExecuteDriver Child');
+let send;
+
+// duplicate defining these keys here so we don't need to re-load a huge appium
+// dependency tree into memory just to run a wdio script
+export const W3C_ELEMENT_KEY = util.W3C_WEB_ELEMENT_IDENTIFIER;
+export const MJSONWP_ELEMENT_KEY = 'ELEMENT';
+
+async function runScript (driverOpts, script, timeout) {
```

Comment:
I usually prefer to ether suffix duration variable names explicitly by the desired measurement unit, for example `timeoutMs` or mention that in the docstring